pkg/controller/storageprofile-controller.go

```diff
@@ -25,6 +26,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+var (
+	// IncompleteProfileGauge is the metric we use to alert about incomplete storage profiles
+	IncompleteProfileGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "kubevirt_cdi_incomplete_storageprofiles_total",
```

Comment:
now as we start seeing the metrics in a few places, we might need a naming util, or at least we can define a kubevirt_cdi as a constant